### PR TITLE
[zh-cn] Update 'image types'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

- [x] upstream: https://github.com/mdn/content/pull/30039

### CODE REVIEW

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 1]
```diff
@@ -5,38 +5,162 @@ slug: Web/Media/Formats/Image_types
...
+      </td>
+    </tr>
+    <tr>
+      <th scope="row"><a href="#AVIF_图像">AVIF</a></th>
+      <th scope="row">AV1 图像档案格式</th>
+      <td><code>image/avif</code></td>
+      <td><code>.avif</code></td>
+      <td>
+        <p>
+          由于其性能高，且无需版税，是图像和动画图像的理想选择。与 PNG 或 JPEG 相比，它的压缩效果更好，支持更高的色深、动画帧、透明度等。请注意，在使用 AVIF 时，应将回退功能加入到浏览器支持更好的格式中（也就是说，要使用 <code><a href="/zh-CN/docs/Web/HTML/Element/picture">&#x3C;picture></a></code> 元素）。<br />
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:44]:
```suggestion
          由于其性能高，且无需版税，是图像和动画图像的理想选择。与 PNG 或 JPEG 相比，它的压缩效果更好，支持更高的色深、动画帧、透明度等。请注意，在使用 AVIF 时，应包含浏览器支持更好的回退格式（也就是说，要使用 <code><a href="/zh-CN/docs/Web/HTML/Element/picture">&#x3C;picture></a></code> 元素）。<br />
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 2]
```diff
@@ -5,38 +5,162 @@ slug: Web/Media/Formats/Image_types
...
+        </p>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row"><a href="#GIF_图像互换格式">GIF</a></th>
+      <th scope="row">图像互换格式</th>
+      <td><code>image/gif</code></td>
+      <td><code>.gif</code></td>
+      <td>
+        是简单图像<em>和</em>动画的不错选择。无损和有索引的静态图像首选 PNG，动画序列可考虑 WebP、AVIF 或 APNG。<br />
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:55]:
```suggestion
        是简单图像和动画的不错选择。无损和有索引的静态图像首选 PNG，动画序列可考虑 WebP、AVIF 或 APNG。<br />
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 3]
```diff
@@ -130,33 +243,122 @@ APNG is ideal for basic animations that do not need to synchronize to other acti
...
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### AVIF 图像
+
+AV1 图像文件格式（AVIF）是一种功能强大、开源、免版税的文件格式，它在高效图像文件格式（HEIF）容器中对 AV1 比特流进行编码。
+
+> **备注：** AVIF 有可能成为在 web 内容中共享图像的“下一件大事”。它具有最先进的功能和性能，却没有阻碍同类替代产品发展的复杂许可和专利使用费的束缚。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:261]:
```suggestion
> **备注：** AVIF 有可能成为在 web 内容中共享图像的“下一件大事”。它具有最先进的特性和性能，却没有阻碍同类替代产品发展的复杂许可和专利使用费的束缚。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 4]
```diff
@@ -130,33 +243,122 @@ APNG is ideal for basic animations that do not need to synchronize to other acti
...
+- 无损压缩。
+- 动画/多图像存储（类似于 GIF 动画，但压缩效果更好）
+- 支持 Alpha 通道（即透明）。
+- _高动态范围（HDR）_：支持存储图像，使图像最亮和最暗部分之间的对比度更大。
+- 宽色域：支持包含更大色彩范围的图像。
+
+AVIF 不支持渐进式渲染，因此文件必须完全下载后才能显示。这通常对真实世界的用户体验影响不大，因为 AVIF 文件比同等的 JPEG 或 PNG 文件小得多，因此下载和显示的速度也快得多。文件大小越大，影响就越大，因此应考虑使用支持渐进式渲染的格式。
+
+Chrome、Opera、Safari 和 Firefox 支持 AVIF（Firefox 支持静态图像，但不支持动画），但 Edge 和 IE 不支持。
+由于支持尚不全面（而且没有什么历史深度），应使用 [`<picture>` 元素](/zh-CN/docs/Web/HTML/Element/picture)（或其他方法）提供 [WebP](#webp_图像)、[JPEG](#jpeg_联合图像专家小组图像) 或 [PNG](#png_便携式网络图形) 格式的后备。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:278]: consisit with the context

```suggestion
Chrome、Opera、Safari 和 Firefox 支持 AVIF（Firefox 支持静态图像，但不支持动画），但 Edge 和 IE 不支持。由于支持尚不全面（而且没有什么历史深度），应使用 [`<picture>` 元素](/zh-CN/docs/Web/HTML/Element/picture)（或其他方法）提供 [WebP](#webp_图像)、[JPEG](#jpeg_联合图像专家小组图像) 或 [PNG](#png_便携式网络图形) 格式的回退。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 5]
```diff
@@ -130,33 +243,122 @@ APNG is ideal for basic animations that do not need to synchronize to other acti
...
+        </p>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">浏览器兼容性</th>
+      <td>
+        Chrome 85、Opera 71、Firefox 93 和 Safari 16.1
+        <ul>
+          <li>
+            Firefox 93 支持静态图像，色彩空间支持全色和有限范围色彩，图像变换支持镜像和旋转。偏好设置 <a href="/zh-CN/docs/Mozilla/Firefox/Experimental_features#avif_compliance_strictness">image.avif.compliance_strictness</a> 可用于调整符合规范的严格程度。不支持动态图像。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:306]:
```suggestion
            Firefox 93 支持静态图像，色彩空间支持全色和有限范围色彩，图像变换支持镜像和旋转。偏好设置 <a href="/zh-CN/docs/Mozilla/Firefox/Experimental_features#avif_严格合规性">image.avif.compliance_strictness</a> 可用于调整符合规范的严格程度。不支持动态图像。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 6]
```diff
@@ -130,33 +243,122 @@ APNG is ideal for basic animations that do not need to synchronize to other acti
...
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">压缩</th>
+      <td>有损压缩和无损压缩</td>
+    </tr>
+    <tr>
+      <th scope="row">授权</th>
+      <td>
+        免版税。有关许可信息，请访问 http://aomedia.org/license/ 。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:345]: could we use `<a>` tag?

**Review by yin1999 — COMMENTED** [review]
reviewed to line 455

File: files/zh-cn/web/media/formats/image_types/index.md (hunk 6 above)
Comment (jasonren0403) [reply files/zh-cn/web/media/formats/image_types/index.md:344]: sure, #mdn/content/pull/30227 is proposed to change en-US content

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 7]
```diff
@@ -293,28 +474,23 @@ GIF is a good choice for simple images and animations, although converting full
...
       <td>
-        All versions of Chrome, Edge, Firefox, Internet Explorer, Opera, and
-        Safari
+        Chrome、Edge、Firefox、Internet Explorer、Opera 和 Safari 浏览器的所有版本
       </td>
     </tr>
     <tr>
-      <th scope="row">最高分辨率</th>
+      <th scope="row">最大尺寸</th>
       <td>65,536×65,536 pixels</td>
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:494]:
```suggestion
      <td>65,536×65,536 像素</td>
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 8]
```diff
@@ -324,77 +500,73 @@ GIF is a good choice for simple images and animations, although converting full
...
 
-An ICO file can contain multiple icons, and begins with a directory listing details about each. Following the directory comes the data for the icons. Each icon's data can be either a [BMP](#bmp_bitmap_file) image without the file header, or a complete [PNG](#png_portable_network_graphics) image (including the file header). If you use ICO files, you should use the BMP format, as support for PNG inside ICO files wasn't added until Windows Vista and may not be well supported.
+一个 ICO 文件可以包含多个图标，文件开头有一个目录，列出每个图标的详细信息。目录之后是图标的数据。每个图标的数据既可以是不带文件头的 [BMP](#bmp_位图文件) 图像，也可以是完整的 [PNG](#png_便携式网络图形) 图像（包括文件头）。如果使用 ICO 文件，则应使用 BMP 格式，因为直到 Windows Vista 才添加了对 ICO 文件内的 PNG 的支持，而且这种支持可能并不完善。
 
-> **警告：** ICO files _should not_ be used in web content. Additionally, their use for favicons has subsided in favor of using a PNG file and the {{HTMLElement("link")}} element, as described in [Providing icons for different usage contexts](/zh-CN/docs/Web/HTML/Element/link#提供用于不同用法上下文的图标).
+> **警告：** ICO 文件*不应*用于网页内容。此外，如[为不同的使用环境提供图标](/zh-CN/docs/Web/HTML/Element/link#提供用于不同用法上下文的图标)所述，它们在收藏夹中的使用已经减少，转而使用 PNG 文件和 {{HTMLElement("link")}} 元素。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:558]:
```suggestion
> **警告：** ICO 文件*不应*用于网页内容。此外，如[为不同的使用环境提供图标](/zh-CN/docs/Web/HTML/Element/link#提供用于不同用法上下文的图标)所述，它们作为网站图标的使用已经减少，转而使用 PNG 文件和 {{HTMLElement("link")}} 元素。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 9]
```diff
@@ -404,123 +576,104 @@ An ICO file can contain multiple icons, and begins with a directory listing deta
...
       <td>
-        All versions of Chrome, Edge, Firefox, Internet Explorer, Opera, and
-        Safari
+        Chrome、Edge、Firefox、Internet Explorer、Opera 和 Safari 浏览器的所有版本
       </td>
     </tr>
     <tr>
-      <th scope="row">最高分辨率</th>
+      <th scope="row">最大尺寸</th>
       <td>256×256 pixels</td>
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:584]:
```suggestion
      <td>256×256 像素</td>
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 10]
```diff
@@ -790,53 +926,36 @@ PNG is widely supported, with all major browsers offering full support for its f
...
-        <a href="http://www.w3.org/Consortium/Legal/copyright-documents"
-          >document use</a
-        >
-        and
-        <a href="http://www.w3.org/Consortium/Legal/copyright-software"
-          >software licensing</a
-        >
-        rules apply. No known royalty-bearing patents.
+        ©2003 <a href="https://www.w3.org/">W3C</a>（<a href="http://www.lcs.mit.edu/">MIT</a>、<a href="http://www.ercim.org/">ERCIM</a>、<a href="https://www.keio.ac.jp/">Keio</a>)，保留所有权利。W3C
+        <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任</a>、<a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">商标</a>、<a href="https://www.w3.org/Consortium/Legal/copyright-documents">文档使用</a>和<a href="https://www.w3.org/Consortium/Legal/copyright-software">软件授权</a>适用。没有已知的专利使用费。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:935]: Please check the upstream

```suggestion
        <a href="https://www.w3.org/policies/#disclaimers">免责声明</a>、<a href="https://www.w3.org/policies/#trademarks">商标</a>、<a href="https://www.w3.org/copyright/document-license/">文档使用</a>和<a href="https://www.w3.org/copyright/software-license/">软件授权</a>适用。没有已知的专利使用费。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 11]
```diff
@@ -869,99 +988,81 @@ SVG is an ideal choice for images which can be represented using a series of dra
...
-        <a href="http://www.w3.org/Consortium/Legal/copyright-documents"
-          >document use</a
-        >
-        and
-        <a href="http://www.w3.org/Consortium/Legal/copyright-software"
-          >software licensing</a
-        >
-        rules apply. No known royalty-bearing patents.
+        >、<a href="http://www.ercim.org/">ERCIM</a>、<a href="https://www.keio.ac.jp/">Keio</a>、<a href="https://ev.buaa.edu.cn/">Beihang</a>），保留所有权利。W3C
+        <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任</a>、<a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">商标</a>、<a href="https://www.w3.org/Consortium/Legal/copyright-documents">文档使用</a>和<a href="https://www.w3.org/Consortium/Legal/copyright-software">软件授权</a>适用。没有已知的专利使用费。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:1037]: +1

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 12]
```diff
@@ -1070,27 +1150,25 @@ Long ago, some browsers supported TIFF images in web content; today, however, yo
...
+WebP 支持通过基于 VP8 视频编解码器的预测编码进行有损压缩，以及通过替换重复数据进行无损压缩。有损 WebP 图像比视觉上压缩级别相似的 JPEG 图像平均小 25-35%。无损 WebP 图像通常比 PNG 格式的相同图像小 26%。
 
-WebP also supports animation: in a lossy WebP file, the image data is represented by a VP8 bitstream, which may contain multiple frames. Lossless WebP holds the `ANIM` chunk, which describes the animation, and the `ANMF` chunk, which represents a frame of an animation sequence. Looping is supported.
+WebP 还支持动画：在有损 WebP 文件中，图像数据由 VP8 比特流表示，其中可能包含多个帧。无损 WebP 包含描述动画的 `ANIM` 块和表示动画序列一帧的 `ANMF` 块。支持循环播放。
 
-WebP now has broad support in the latest versions of major web browsers, although it does not have deep historical support. Provide a fallback in either [JPEG](#jpeg_joint_photographic_experts_group_image) or [PNG](#png_portable_network_graphics) format, such as with [the `<picture>` element](/zh-CN/docs/Web/HTML/Element/picture).
+WebP 目前已在主要网络浏览器的最新版本中获得广泛支持，但还没有深入的历史支持。需要提供 [JPEG](#jpeg_联合图像专家小组图像) 或 [PNG](#png_便携式网络图形) 格式的后备，例如 [`<picture>` 元素](/zh-CN/docs/Web/HTML/Element/picture)。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:1171]:
```suggestion
WebP 目前已在主要网络浏览器的最新版本中获得广泛支持，但还没有深入的历史支持。需要提供 [JPEG](#jpeg_联合图像专家小组图像) 或 [PNG](#png_便携式网络图形) 格式的回退，例如 [`<picture>` 元素](/zh-CN/docs/Web/HTML/Element/picture)。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 13]
```diff
@@ -1265,80 +1338,116 @@ static unsigned char square8_bits[] = {
...
+</table>
 
-### Icons
+### 图标
 
-For smaller images such as icons, use a lossless format to avoid loss of detail in a size-constrained image. While lossless WebP is ideal for this purpose, support is not widespread yet, so PNG is a better choice unless you offer a [fallback](#提供后备图像). If your image contains fewer than 256 colors, GIF is an option, although PNG often compresses even smaller with its indexed compression option (PNG-8).
+对于图标等较小的图像，应使用无损格式，以避免在大小受限的图像中丢失细节。虽然无损 WebP 是实现这一目的的理想选择，但其支持尚未普及，因此 PNG 是更好的选择，除非你提供[后备选项](#提供后备图像)。如果图像包含的颜色少于 256 种，GIF 也是一种选择，不过 PNG 通过其索引压缩选项（PNG-8）通常可以压缩得更小。
 
-If the icon can be represented using vector graphics, consider [SVG](#svg_scalable_vector_graphics), since it scales across various resolutions and sizes, so it's perfect for responsive design. Although SVG support is good, it may be worth offering a PNG fallback for older browsers.
+如果图标可以用矢量图形表示，可以考虑使用 [SVG](#svg_可缩放矢量图形)，因为它可以在各种分辨率和尺寸下缩放，非常适合响应式设计。虽然 SVG 支持良好，但仍值得为旧版浏览器提供 PNG 后备功能。
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:1377]: align the translation of "fallback": "后备", "回退", (there are more than one occurrences)

**Review by yin1999 — COMMENTED** [review]
final review

File: files/zh-cn/web/media/formats/image_types/index.md (hunk 6 above)
Comment (yin1999) [reply files/zh-cn/web/media/formats/image_types/index.md:344]: For licensing, I'd prefer "许可"

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 14]
```diff
@@ -5,38 +5,162 @@ slug: Web/Media/Formats/Image_types
...
+      <th scope="row">文件格式</th>
+      <th scope="col">MIME 类型</th>
+      <th scope="col">文件扩展名</th>
+      <th scope="col">摘要</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">
+        <a href="#apng_动态可移植网络图形">APNG</a>
```
Comment (yin1999) [inline files/zh-cn/web/media/formats/image_types/index.md:27]: Those hashs are broken, please update (with the below occurences)

```suggestion
        <a href="#apng（动态可移植网络图形）">APNG</a>
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 15]
```diff
@@ -130,33 +243,121 @@ APNG is ideal for basic animations that do not need to synchronize to other acti
...
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">压缩</th>
+      <td>有损压缩和无损压缩</td>
+    </tr>
+    <tr>
+      <th scope="row">授权</th>
+      <td>
+        免版税。有关许可信息，请访问 http://aomedia.org/license/ 。
```
Comment (jasonren0403) [inline files/zh-cn/web/media/formats/image_types/index.md:344]:
```suggestion
        免版税。有关许可信息，请访问<a href="https://aomedia.org/license/">版权页面</a>。
```

File: files/zh-cn/web/media/formats/image_types/index.md (hunk 11 above)
Comment (jasonren0403) [reply files/zh-cn/web/media/formats/image_types/index.md:1037]:
```suggestion
        <a href="https://www.w3.org/policies/#disclaimers">责任</a>、<a href="https://www.w3.org/policies/#trademarks">商标</a>、<a href="https://www.w3.org/copyright/document-license/">文档使用</a>和<a href="https://www.w3.org/copyright/software-license/">软件授权</a>适用。没有已知的专利使用费。
```

File: files/zh-cn/web/media/formats/image_types/index.md [hunk 16]
```diff
@@ -1131,66 +1203,67 @@ WebP now has broad support in the latest versions of major web browsers, althoug
...
     <tr>
       <th scope="row">授权</th>
-      <td>No license required; source code is openly available.</td>
+      <td>无需许可证；源代码可公开获取。</td>
     </tr>
   </tbody>
 </table>
 
-### <a id="XBM" name="XBM">XBM</a> (X Window System Bitmap file)
+> **备注：** 尽管 Apple [宣布在 Safari 14 中支持 WebP](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174)，但在 Safari 版本 16.0 之前，`.webp` 图像无法在早于 Big Sur 版本的 macOS 桌面版本上正常显示，而 iOS 14 上的 Safari *却能*正常显示 `.webp` 图像。
```
Comment (jasonren0403) [inline files/zh-cn/web/media/formats/image_types/index.md:1258]: wait for https://github.com/mdn/content/pull/30039/files to decide what content should be written here

Comment (jasonren0403) [reply files/zh-cn/web/media/formats/image_types/index.md:1258]:
```suggestion
> **备注：** 在 macOS 版 Safari 上，WebP 支持取决于 Safari 和 macOS 的版本。你需要 Safari 14 或更高版本，以及 macOS Big Sur (11) 或更高版本。
```

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**